src/web-ui/.env

```diff
@@ -1,36 +1,27 @@
-# This file is automatically generated by gen_env.sh during the build stage 
-# in CodePipeline. It is also used directly when running the web-ui locally 
-# in a Docker container. When running the web-ui locally, update the values 
-# below to match your configuration.
-
-# Configure Dependent Services
-VUE_APP_PRODUCTS_SERVICE_DOMAIN=http://localhost
-VUE_APP_PRODUCTS_SERVICE_PORT=8001
-VUE_APP_USERS_SERVICE_DOMAIN=http://localhost
-VUE_APP_USERS_SERVICE_PORT=8002
-VUE_APP_CARTS_SERVICE_DOMAIN=http://localhost
-VUE_APP_CARTS_SERVICE_PORT=8003
-VUE_APP_ORDERS_SERVICE_DOMAIN=http://localhost
-VUE_APP_ORDERS_SERVICE_PORT=8004
-VUE_APP_RECOMMENDATIONS_SERVICE_DOMAIN=http://localhost
-VUE_APP_RECOMMENDATIONS_SERVICE_PORT=8005
-VUE_APP_SEARCH_SERVICE_DOMAIN=http://localhost
-VUE_APP_SEARCH_SERVICE_PORT=9200
-
-# Configure Authentication
-VUE_APP_AWS_REGION=us-west-2
+VUE_APP_PRODUCTS_SERVICE_DOMAIN=
+VUE_APP_PRODUCTS_SERVICE_PORT=80
+VUE_APP_USERS_SERVICE_DOMAIN=
+VUE_APP_USERS_SERVICE_PORT=80
+VUE_APP_CARTS_SERVICE_DOMAIN=
+VUE_APP_CARTS_SERVICE_PORT=80
+VUE_APP_ORDERS_SERVICE_DOMAIN=
+VUE_APP_ORDERS_SERVICE_PORT=80
+VUE_APP_RECOMMENDATIONS_SERVICE_DOMAIN=
+VUE_APP_RECOMMENDATIONS_SERVICE_PORT=80
+VUE_APP_SEARCH_SERVICE_DOMAIN=
+VUE_APP_SEARCH_SERVICE_PORT=80
```

Comment:
Was committing changes to this file an oversight? This will break the default port numbers used in the local Docker testing setup with docker-compose. Suggest backing out changes to this file.

Reply:
Yes, this was a mistake.  I must have missed that in a commit.  Reverted this back to the master .env file.